tests/integration/spark/test_reducer_merge.py

```diff
@@ -0,0 +1,168 @@
+import unittest, PyRDF, ROOT
+
+class ReducerMergeTest(unittest.TestCase):
+    """
+    Integration tests to check the working of merge operations
+    in the reducer function.
+
+    """
+    @classmethod
+    def setUpClass(cls):
+        """
+        Set up method to select Spark backend
+        before running all the tests.
+
+        """
+        PyRDF.use("spark", {'npartitions':2, 'spark.executor.instances':2})
+
+    def assertHistoOrProfile(self, obj_1, obj_2):
+        """
+        Asserts equality between two 'ROOT.TH1' or
+        'ROOT.TH2' objects.
+
+        """
+        # Compare the sizes of equivalent objects
+        self.assertEqual(obj_1.GetEntries(), obj_2.GetEntries())
+
+        # Compare the means of equivalent objects
+        self.assertEqual(obj_1.GetMean(), obj_2.GetMean())
+
+        # Compare the standard deviations of equivalent objects
+        self.assertEqual(obj_1.GetStdDev(), obj_2.GetStdDev())
+
+    def define_two_columns(self, rdf):
+        """
+        Helper method that Defines and returns
+        two columns with definitions "x = tdfentry_"
+        and "y = tdfentry_ * tdfentry_".
+
+        """
+        return rdf.Define("x", "tdfentry_").Define("y", "tdfentry_*tdfentry_")
+
+    def define_three_columns(self, rdf):
+        """
+        Helper method that Defines and returns three columns
+        with definitions "x = tdfentry_", "y = tdfentry_ * tdfentry_"
+        and "z = tdfentry_ * tdfentry_ * tdfentry_".
+
+        """
+        return rdf.Define("x", "tdfentry_").Define("y", "tdfentry_*tdfentry_").Define("z", "tdfentry_*tdfentry_*tdfentry_")
+
+    def test_histo1d_merge(self):
+        """
+        Integration test to check the working of
+        Histo1D merge operation in the reducer.
+
+        """
+        # Operations with PyRDF
+        rdf_py = PyRDF.RDataFrame(10)
```

Comment:
What about refactoring a bit more:
```python
rdf_py, rdf_cpp = self.create_test_rdfs()
```
Same below.

Reply:
But @etejedor , wouldn't it be more clear if I separate the declaration of `PyRDF` and `PyROOT` variables ?

Reply:
And it also separately defines what I want to test and what I want to assert it with. What do you say ?

Reply:
Ok let's merge